lib/listen/monotonic_time.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Listen
+  module MonotonicTime
+    class << self
+      def now
+        if defined?(Process::CLOCK_MONOTONIC)
+          Process.clock_gettime(Process::CLOCK_MONOTONIC)
```

Comment:
I can tell you that on every Ruby implementation that I'm aware of, `Process::CLOCK_MONOTONIC` will be defined.

If you insist on this, you should at least put the if statement outside the method definition rather than checking it every time.

Reply:
Since I can never see every Ruby implementation, I feel best going with the documentation.

I moved the `if` outside. That was tricky to get the tests to still pass, but I think I got, by using `load` to keep reloading the file, and especially one final time in a `before(:all)` block. https://github.com/guard/listen/pull/512/commits/0ea75950098ff11e1c217119dd44181a8500c6d8